torch/csrc/Exceptions.h

```diff
@@ -127,6 +133,7 @@ static inline void PyErr_SetString(PyObject* type, const std::string& message) {
 #define END_HANDLE_TH_ERRORS END_HANDLE_TH_ERRORS_RET(nullptr)
 
 extern PyObject *THPException_FatalError;
+extern PyObject *THPException_GPUOutOfMemoryError;
```

Comment:
Try accessing this via a function (rather than extern data); e.g., `PyObject* TORCH_API getTHPException_GPUOutOfMemoryError()`, I'm guessing that will shut Windows up.